job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -298,6 +298,8 @@ definitions:
       description:
         type: string
         description: The description of the job
+      expiration-policy:
```

Comment:
Rightly or wrongly the other properties in this object use camelCase, so we wouldn't want a new one to use kebab-case.

Reply:
Done

Reply:
Done